cmd/cliclient/cleanup.go

```diff
@@ -0,0 +1,61 @@
+package cliclient
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ory/x/configx"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ory/kratos/driver"
+	"github.com/ory/kratos/driver/config"
+	"github.com/ory/x/cmdx"
+	"github.com/ory/x/flagx"
+)
+
+type CleanupHandler struct{}
+
+func NewCleanupHandler() *CleanupHandler {
+	return &CleanupHandler{}
+}
+
+func (h *CleanupHandler) CleanupSQL(cmd *cobra.Command, args []string) {
+	var d driver.Registry
+
+	if flagx.MustGetBool(cmd, "read-from-env") {
+		d = driver.NewWithoutInit(
+			cmd.Context(),
+			cmd.ErrOrStderr(),
+			configx.WithFlags(cmd.Flags()),
+			configx.SkipValidation())
+		if len(d.Config(cmd.Context()).DSN()) == 0 {
+			fmt.Println(cmd.UsageString())
+			fmt.Println("")
+			fmt.Println("When using flag -e, environment variable DSN must be set")
```

Comment:
```suggestion
			fmt.Fprintln(cmd.OutOrStdout(), cmd.UsageString())
			fmt.Fprintln(cmd.OutOrStdout(), "")
			fmt.Fprintln(cmd.OutOrStdout(), "When using flag -e, environment variable DSN must be set")
```

Also here we want to use the `out` from the command such that we can in tests write to a buffer and do assertions on the output.